### ISSUE TITLE
Alt attribute

### ISSUE THREAD

**Issue by jim-rosales** [hngo1025/prj-rev-bwfs-dasmoto#3]
When linking images on a site, it is best practice including an alt attribute for each image. The alt attribute is used for [accessibility](https://www.codecademy.com/articles/accessibility) and as a back up in case your image ever fails to load.

https://github.com/hngo1025/prj-rev-bwfs-dasmoto/blob/d9ce0413d9d6d19ef566ef9a61e4c2617f7c216f/Dasmotos/index.html#L17
